mathics/core/symbols.py

```diff
@@ -8,7 +8,6 @@
 
 from mathics.core.element import (
     BaseElement,
-    ImmutableValueMixin,
```

Comment:
We seem to want to import this so that it can be erroneously imported inside atoms.py. Instead atoms.py imports from the place this is really defined.